### PR TITLE
Tokenize Hebrew words as global constants v2.0

### PR DESCRIPTION
**Pull request by shakedzy**
@elazarg @idosavion let me know what you think

### CODE REVIEW

**Review by idosavion — COMMENTED** [review]
Shaked, first of all, thanks for the PR! 😄 
1. I think we should discuss the problem and the way we want to solve it offline to save you some time of work.
2. I'm not sure this is the interface we want: `words.concat(words.ROAD, "4", words.AND(words.IN(words.ROAD)), "7")`, can be a bit annoying.
3. I tried to translate a few of the words I found in the news flash using google translate and managed to get the same interface as you added here, and I don't want to add more code if we're not required to.
4. If we will eventually decide to add this code, we should add some unit tests

**Comment by shakedzy** [comment]
@idosavion so how do you see this? How should we replace all Hebrew words with constants? I thought about having just a `dict`, but then all the IN/THE etc will require extra key-value pairs, which make no sense..

**Comment by elazarg** [comment]
I think that in order to be evaluated, the PR should be augmented with its "client" use, using at least 10 examples from different parts of the source code.

Also, for purposes of i18n, it's better if entire phrases are extracted, possibly with format strings. In principle, I don't like the idea of constructing phrases mechanically; it does not generalize well.

This PR is strictly better than the previous one, but I'd still vote for doing things "the standard way".

File: anyway/language_constants.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
+class Hebrew(object):
+    ROAD = "כביש"
+    INTERCITY = "בינעירוני"
+
+    def __init__(self, *args, **kwargs):
+        super(Hebrew, self).__init__(*args, **kwargs)
```
Comment (elazarg) [inline anyway/language_constants.py:6]: This constructor can be left out. Also, why instantiate this class at all? Even further, use a class at all? A static class is essentially a module.

File: anyway/language_constants.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+    @staticmethod
+    def concat(*words, sep=" "):
+        return sep.join(words)
+
+    @staticmethod
+    def IN(word):
+        return "ב" + word
```
Comment (elazarg) [inline anyway/language_constants.py:14]: Why use non-standard ALL_CAPS?

Comment (shakedzy) [reply anyway/language_constants.py:14]: because `in` is a reserved word

**Comment by shakedzy** [comment]
As the new guy here, I think I'll let you guys think of what is the best way to make this happen rather than deciding myself. I get the feeling I don't see the whole picture, and that makes total sense.. I'll jump back in once there's a clear decision :)

File: anyway/language_constants.py (hunk 2 above)
Comment (elazarg) [reply anyway/language_constants.py:14]: That is not, by itself, enough of a reason. You could use `in_` or `at`.

**Pull request closed without merging**